### PR TITLE
Fixes #5336 - remove Ex#3 - style cleanup

### LINKED ISSUE #5336: Example 3 is broken; could be removed altogether (covered by example 4)
> `Invoke-CimMethod -ClassName Win32_Process -MethodName "Create" -Arguments @{ Path = "notepad.exe" }`

should be (different key name):

```
Invoke-CimMethod -ClassName Win32_Process -MethodName "Create" -Arguments @{ CommandLine = "notepad.exe" }
```

However, given that example 4 pretty much covers what example 3 shows as well, plus an additional argument, example 3 could be removed altogether.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 13b8cf71-09c0-4716-7ca6-121aee18e163
* Version Independent ID: ecde04e2-b970-8c9c-754d-ffc4fbaa973e
* Content: [Invoke-CimMethod (CimCmdlets)](https://docs.microsoft.com/en-us/powershell/module/cimcmdlets/invoke-cimmethod?view=powershell-7)
* Content Source: [reference/7.0/CimCmdlets/Invoke-CimMethod.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/7.0/CimCmdlets/Invoke-CimMethod.md)
* Product: **powershell**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary
Fixes #5336 - remove Ex#3 - style cleanup
Fixes [AB#1667152](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1667152)

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7.x preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/5.1/CimCmdlets/Invoke-CimMethod.md [hunk 1]
```diff
@@ -17,71 +17,81 @@ Invokes a method of a CIM class.
 ## SYNTAX
```
Comment (chasewilson) [inline reference/5.1/CimCmdlets/Invoke-CimMethod.md]: Won't let me place a comment above for something. The Synopsis Header should be surrounded by new lines

File: reference/5.1/CimCmdlets/Invoke-CimMethod.md [hunk 2]
```diff
@@ -340,15 +336,13 @@ Accept wildcard characters: False
...
-A method is invoked on the instances received as a result of the query.
-
-You can specify the query dialect using the QueryDialect parameter.
+Specifies a query to run on the CIM server. A method is invoked on the instances received as a
+result of the query. You can specify the query dialect using the **QueryDialect** parameter.
 
-If the value specified contains double quotes ("), single quotes ('), or a backslash (\\), you must
-escape those characters by prefixing them with the backslash (\\) character. If the value specified
-uses the WQL LIKE operator, then you must escape the following characters by enclosing them in
-square brackets (\[\]): percent (%), underscore (_), or opening square bracket (\[).
```
Comment (chasewilson) [inline reference/5.1/CimCmdlets/Invoke-CimMethod.md]: This should be a comma: (`[]`), not (`[]`):

File: reference/5.1/CimCmdlets/Invoke-CimMethod.md (hunk 1 above)
Comment (sdwheeler) [reply reference/5.1/CimCmdlets/Invoke-CimMethod.md]: No, that is the one exception. If you put a blank line after the Synopsis then the text does not appear in Get-Help.

File: reference/5.1/CimCmdlets/Invoke-CimMethod.md (hunk 2 above)
Comment (sdwheeler) [reply reference/5.1/CimCmdlets/Invoke-CimMethod.md]: No. It is saying that you need to put the following characters inside square brackets: `%`, `_`, `[`.

Comment (DCtheGeek) [reply reference/5.1/CimCmdlets/Invoke-CimMethod.md]: I made the same misread, thanks for clarifying.

**Pull request merged**